frontend/src/app/admin/application.service.ts

```diff
@@ -11,13 +11,16 @@ import { Application } from './application';
 @Injectable()
 export class ApplicationService {
 
-  private endpoint = 'https://fs-intake-api-staging.app.cloud.gov/permits/applications';
-//  private endpoint = 'http://localhost:8080/permits/applications';
+//  private endpoint = 'https://fs-intake-api-staging.app.cloud.gov/permits/applications';
+  private endpoint = 'http://localhost:8080/permits/applications';
```

Comment:
Can we update the base URL to use an environment variable instead of commenting things out?